uPortal-soffit-renderer/src/main/java/org/apereo/portal/soffit/renderer/SoffitRendererController.java

```diff
@@ -123,8 +121,10 @@ public ModelAndView render(final HttpServletRequest req, final HttpServletRespon
         // Set up cache headers appropriately
         configureCacheHeaders(res, module);
 
-        return new ModelAndView(viewName.toString(), PORTAL_REQUEST_MODEL_NAME, portalRequest);
+        ModelAndView mav = new ModelAndView(viewName.toString());
```

Comment:
After talking with @bpowell. It seems like him and @drewwills  have come to the conclusion of going a different route. Therefore, I'll be closing this request.